docker-compose/cassandra-3.11/README.md

```diff
@@ -1,27 +1,38 @@
-This directory provides two ways to start Stargate with Cassandra 3.11 using docker-compose.
+# Docker Compose scripts for Stargate with Cassandra 3.11 
```

Comment:
Do you think it would be valuable to add a debugging info like this one to all readmes:

```
## Debugging

If you want to debug a docker container, add following debug arg to the `JAVA_OPTS` of the target container:

`-agentlib:jdwp=transport=dt_socket,address=host.docker.internal:8000,server=n,suspend=n`

Then create a `LIsten to remote JVM` debugging run configuration in IntelliJ that points to the port 8000.
First start the debugger and then hit `docker-compose up`.
```

Well, this is for coordinator mainly, not sure anymore